chapters/io.rst

```diff
@@ -0,0 +1,277 @@
+How to handle Input and Output for Flows
+=======================================
+
+This chapter explains how input and output data is handled by the `Flow API`.
+
+Input
+-----
+The input data for the `Flow` functions `flow.index(...)`, `flow.update(...)` and `flow.search(...)` can be provided in three different ways.
```

Comment:
Note `` `italic` `` is for italic in rst. Please use either `` :term:`parameter ` `` or `` `` `` for verbatim 
```suggestion
The input data for the ``Flow`` functions ``flow.index(...)``, ``flow.update(...)`` and ``flow.search(...)`` can be provided in three different ways.
```

Reply:
thanks - I changed all ` to ``